Bottom: 7c7253fbda963ab279336f08306bfb987ff84d79
Top:    daf53db464152f40dc8d6f2c99844510b03f8567
Author: Roland Dreier <roland@purestorage.com>
Date:   2012-10-16 19:44:39 +0000

rdma_xserver/client: Fix man page formatting

Putting 'r' at the beginning of a line in the nroff source for man pages
is confusing to nroff because lines that start with a single quote
character ' or a dot character . are treated as control lines, which is
not what's intended here.  Some of the man page text ends up left out of
the formatted output.

Fix this by just wrapping the text slightly differently in the source
(which doesn't matter since nroff reflows the text anyway).  Also add a
missing ".TP" so that the -p and -c options are not run together in the
formatted output.

Signed-off-by: Roland Dreier <roland@purestorage.com>
Signed-off-by: Sean Hefty <sean.hefty@intel.com>


---

diff --git a/man/rdma_xclient.1 b/man/rdma_xclient.1
index 087e7fe..b4bb045 100644
--- a/man/rdma_xclient.1
+++ b/man/rdma_xclient.1
@@ -20,9 +20,10 @@ By default, the client will attempt to connect to the server using
 \-p server_port
 Specifies the port number that the server listens on.  By default the server
 listens on port 7471.
+.TP
 \-c communication type
-Specifies the type of communication established with the server program.
-'r' results in using a reliable-connected QP (the default).  'x' uses
+Specifies the type of communication established with the server program.  'r'
+results in using a reliable-connected QP (the default).  'x' uses
 extended reliable-connected XRC QPs.
 .SH "NOTES"
 Basic usage is to start rdma_xserver, then connect to the server using the
diff --git a/man/rdma_xserver.1 b/man/rdma_xserver.1
index 75529e4..a26c962 100644
--- a/man/rdma_xserver.1
+++ b/man/rdma_xserver.1
@@ -14,9 +14,10 @@ data.
 \-p port
 Changes the port number that the server listens on.  By default the server
 listens on port 7471.
+.TP
 \-c communication type
-Specifies the type of communication established with the client program.
-'r' results in using a reliable-connected QP (the default).  'x' uses
+Specifies the type of communication established with the client program.  'r'
+results in using a reliable-connected QP (the default).  'x' uses
 extended reliable-connected XRC QPs.
 .SH "NOTES"
 Basic usage is to start rdma_xserver, then connect to the server using the
